app/dashboard/page.tsx

```diff
@@ -28,6 +31,39 @@ export default function Dashboard() {
 
         const data = await response.json();
         setGiftExchanges(data);
+
+        const today = new Date();
+        for (const exchange of data) {
+          const drawingDate = new Date(exchange.drawing_date);
+          const timeDifference = drawingDate.getTime() - today.getTime();
+          const dayDifference = Math.ceil(
+            timeDifference / (1000 * 60 * 60 * 24),
+          );
+
+          if (dayDifference > 0 && dayDifference <= 3) {
+            toast({
+              variant: ToastVariants.Warning,
+              title: `Upcoming Draw - ${exchange.name}`,
+              description: `The draw is in ${dayDifference} day${dayDifference < 2 ? '' : 's'}!`,
```

Comment:
Instead of calculating this, just make the string 
```suggestion
              description: `The draw is in ${dayDifference} day(s)!`,
```